apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/edit.js

```diff
@@ -134,24 +121,20 @@ export default compose( [
 	withColors( 'backgroundColor', { textColor: 'color' } ),
 	withFontSizes( 'fontSize' ),
 	withSelect( ( select, { clientId } ) => {
-		const { isSavingPost, isPublishingPost, isAutosavingPost, isCurrentPostPublished } = select(
-			'core/editor'
-		);
 		const { getBlockIndex, getBlockRootClientId, getTemplateLock } = select( 'core/block-editor' );
 		const rootClientId = getBlockRootClientId( clientId );
 
 		return {
 			blockIndex: getBlockIndex( clientId, rootClientId ),
 			isLocked: !! getTemplateLock( rootClientId ),
 			rootClientId,
-			shouldUpdateSiteOption:
-				( ( isSavingPost() && isCurrentPostPublished() ) || isPublishingPost() ) &&
-				! isAutosavingPost(),
 		};
 	} ),
 	withDispatch( ( dispatch, { blockIndex, rootClientId } ) => ( {
-		createErrorNotice: dispatch( 'core/notices' ).createErrorNotice,
 		insertDefaultBlock: () =>
```

Comment:
btw @Copons while we're looking at this code, do you know what `insertDefaultBlock` is for? I don't really get why we need to specify anything here

Reply:
It's used in the `RichText.onReplace`: https://github.com/Automattic/wp-calypso/blob/9cea972108ef03bcca545c81a8359709a2ed5cf1/apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/edit.js#L105

It's used to handle line breaks: instead of breakng inside the component, it adds a new (default) block after.

Reply:
ah I see :) thanks for the clarification!